inc/form.class.php

```diff
@@ -1296,15 +1296,18 @@ public function duplicate() {
             return false;
          }
 
+         $new_target_item_id = $target->getField('items_id');
          switch ($target_values['itemtype']) {
             case PluginFormcreatorTargetTicket::class:
+               // Get the original target ticket
                if (!$target_ticket->getFromDB($target_values['items_id'])) {
                   return false;
                }
 
                // Update the target ticket created while cloning the target
                $update_target_ticket = $target_ticket->fields;
-               unset($update_target_ticket['id'], $update_target_ticket['uuid']);
+               $update_target_ticket['id'] = $new_target_item_id;
```

Comment:
this is the source target ticket, we update the ID with the ID of the duplicated one, available in the duplicated isntance of Target.